markdown/docs/addon-author-guide.md

```diff
@@ -37,19 +38,21 @@ Node, this value is not set.
 
 ### Tracking It Down
 
-FastBoot works by turning an Ember application and all of its
-dependencies, including your addon, into a single bundle of JavaScript.
-Similar to how you ship an `app.js` and `vendor.js` file to the browser
-when you deploy an Ember app, the FastBoot server also loads these two
-files.
+FastBoot works by running a singular build as it was running without the FastBoot
+addon being installed. It basically loads the same assets `app.js` and `vendor.js`
```

Comment:
Sounds much better :). Just tweaked the "server and browser clients" to call "browser and Node" since everywhere else we refer it as Node.